extensions/amp-bind/amp-bind.md

```diff
@@ -313,97 +313,102 @@ Only binding to the following components and attributes are allowed:
     <th>Behavior</th>
   </tr>
   <tr>
-    <td class="col-thirty"><code>&lt;amp-brightcove></code></td>
+    <td class="col-thirty"><code>&lt;amp-brightcove&gt;</code></td>
     <td class="col-fourty"><code>[data-account]</code><br><code>[data-embed]</code><br><code>[data-player]</code><br><code>[data-player-id]</code><br><code>[data-playlist-id]</code><br><code>[data-video-id]</code></td>
     <td class="col-thirty">Changes the displayed Brightcove video.</td>
   </tr>
   <tr>
-    <td><code>&lt;amp-carousel type=slides></code></td>
+    <td><code>&lt;amp-carousel type=slides&gt;</code></td>
     <td><code>[slide]</code><sup>1</sup></td>
     <td>Changes the currently displayed slide index. <a href="https://ampbyexample.com/advanced/image_galleries_with_amp-carousel/#linking-carousels-with-amp-bind">See an example</a>.</td>
   </tr>
   <tr>
-    <td><code>&lt;amp-iframe></code></td>
+    <td><code>&lt;amp-iframe&gt;</code></td>
     <td><code>[src]</code></td>
     <td>Changes the iframe's source URL.</td>
   </tr>
   <tr>
-    <td><code>&lt;amp-img></code></td>
+    <td><code>&lt;amp-img&gt;</code></td>
     <td><code>[alt]</code><br><code>[attribution]</code><br><code>[src]</code><br><code>[srcset]</code></td>
     <td>See corresponding <a href="https://www.ampproject.org/docs/reference/components/media/amp-img#attributes">amp-img attributes</a>.</td>
   </tr>
   <tr>
-    <td><code>&lt;amp-list></code></td>
+    <td><code>&lt;amp-list&gt;</code></td>
     <td><code>[src]</code></td>
     <td>Fetches JSON from the new URL and re-renders, replacing old content.</td>
   </tr>
   <tr>
-    <td><code>&lt;amp-selector></code></td>
+    <td><code>&lt;amp-list&gt;</code></td>
```

Comment:
@bpaduch Is there a way to merge this cell with the one above? I.e. does `rowspan` work here?